doc/nrf/libraries/modem/at_custom_cmd.rst

```diff
@@ -0,0 +1,114 @@
+.. _at_custom_cmd_readme:
+
+Custom AT commands library
+##########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The custom AT commands library allows the application to register AT commands with the Modem library that is handled in the application. This allows the application to add custom AT commands, and to override modem AT calls with application callbacks.
+
+To add a custom AT command or overwrite an existing command, the AT commands filter and callback must be added with the :c:macro:`AT_FILTER` macro in :file:`at_custom_cmd.h`.
+If the filter matches the start of the command sent to :c:func:`nrf_modem_at_cmd`, or the command in its entirety, the AT command is passed to the filter callback instead of to the modem.
+When returning from the callback, the content of the provided buffer is treated as the modem response.
+
+.. note::
+When the custom AT commands library is enabled, the application must not call :c:func:`nrf_modem_at_filter_set` as this will override the handler set by the library.
```

Comment:
```suggestion
.. note::
   When the custom AT commands library is enabled, the application must not call :c:func:`nrf_modem_at_filter_set` as this will override the handler set by the library.
```